api/authenticator.py

```diff
@@ -202,6 +211,11 @@ def __init__(self,
         # to have it available for notifications.
         self.email_address = email_address
 
+        # We do not store the patron's neighborhood in the database
+        # record, but we may need it to store in the database records
+        # for circulation events triggered by this patron.
+        self.neighborhood = neighborhood
+
```

Comment:
I felt a need to explain every that this is not going into the database every time there was an assignment that might cause someone scanning the code to be alarmed.